### PR TITLE
remove element directly from target

### PR DESCRIPTION
**Pull request by lukasIO**
Hi,

for me the content of the portal never got actually removed because the parent was always undefined. This approach seems to fix it.

### CODE REVIEW

**Comment by romkor** [comment]
Hi @lukasIO!
Can you share some example on https://svelte.dev/repl?

**Comment by lukasIO** [comment]
Hi @romkor,

should have specified the scenario a bit, sorry. 
It happens for me when using sapper and the portal target is part of a _layout route while the portal is being called from a "regular" component.

I'll see if I can spin up a minimal repro repo.

**Comment by romkor** [comment]
Please create issue with detailed description what is broken. And create reproducible example on https://svelte.dev/repl.
I will close this pull request.

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "svelte-portal",
+  "name": "@neesh-studio/svelte-portal",
```
Comment (romkor) [inline package.json:2]: 😎

Comment (lukasIO) [reply package.json:2]: sorry, forgot about the PR still being open, we had to fix the problem in our codebase - that should of course never go into this PR however.

**Pull request closed without merging**